assets/styles/style.css

```diff
@@ -436,3 +436,12 @@ PLUGIN STYLES
       grid-template-areas: 'Logo TopMenu User' 'Logo SubMenu User';
   }
 }
+
+.hr {
```

Comment:
Note that since this is `.hr` instead of `hr` you're styling a thing with a class of `hr` rather than any hr tags.